src/styles/SearchForm.css

```diff
@@ -0,0 +1,44 @@
+.search-form {
+  margin: 0px;
+  width: 50%;
+  height: 40px;
+  display: inline-block;
+}
+
+.search-form>input {
```

Comment:
The attention to detail to get the sort button to fit seamlessly into the search field is great.

I think there may be an easier way to handle the border radius. This CSS is great but always try and work on your CSS and make it cleaner and more readable. Clean code applies to CSS just as much as javascript